2022-10-11,22:00:20 [INFO] (app.py:app:244): Parsing config file C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tabpy_server\app\..\common\default.conf
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter port set to "9004" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter server_version set to "2.5.0" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter evaluate_enabled set to "True" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter evaluate_timeout set to "30" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter upload_dir set to "C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tmp\query_objects" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter transfer_protocol set to "http" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:212): Parameter certificate_file is not set
2022-10-11,22:00:20 [DEBUG] (app.py:app:212): Parameter key_file is not set
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter state_file_path set to "C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tabpy_server" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter static_path set to "C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tabpy_server\static" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:212): Parameter TABPY_PWD_FILE is not set
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter log_request_context set to "false" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:205): Parameter max_request_size_in_mb set to "100" from default value
2022-10-11,22:00:20 [DEBUG] (app.py:app:430): File C:\Users\ACER\anaconda3\envs\tetris\Lib\site-packages\tabpy\tabpy_server\state.ini not found, creating from template C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tabpy_server\state.ini.template...
2022-10-11,22:00:20 [INFO] (app.py:app:436): Loading state from state file C:\Users\ACER\anaconda3\envs\tetris\Lib\site-packages\tabpy\tabpy_server\state.ini
2022-10-11,22:00:20 [DEBUG] (app.py:app:313): Static pages folder set to "C:\Users\ACER\anaconda3\envs\tetris\lib\site-packages\tabpy\tabpy_server\static"
2022-10-11,22:00:20 [INFO] (app.py:app:332): Password file is not specified: Authentication is not enabled
2022-10-11,22:00:20 [INFO] (app.py:app:347): Call context logging is disabled
2022-10-11,22:00:20 [INFO] (app.py:app:125): Initializing TabPy...
2022-10-11,22:00:20 [DEBUG] (proactor_events.py:proactor_events:626): Using proactor: IocpProactor
2022-10-11,22:00:20 [INFO] (callbacks.py:callbacks:43): Initializing TabPy Server...
2022-10-11,22:00:20 [DEBUG] (state.py:state:613): Loading option 'None' from section [Query Objects Service Versions]...
2022-10-11,22:00:20 [DEBUG] (state.py:state:635): Returning value '[]'
2022-10-11,22:00:20 [DEBUG] (state.py:state:148): Collected endpoints: {}
2022-10-11,22:00:20 [INFO] (app.py:app:129): Done initializing TabPy.
2022-10-11,22:00:20 [DEBUG] (selector_events.py:selector_events:54): Using selector: SelectSelector
2022-10-11,22:00:20 [INFO] (app.py:app:83): Setting max request size to 104857600 bytes
2022-10-11,22:00:20 [INFO] (callbacks.py:callbacks:64): Initializing models...
2022-10-11,22:00:20 [DEBUG] (state.py:state:613): Loading option 'None' from section [Query Objects Service Versions]...
2022-10-11,22:00:20 [DEBUG] (state.py:state:635): Returning value '[]'
2022-10-11,22:00:20 [DEBUG] (state.py:state:148): Collected endpoints: {}
2022-10-11,22:00:20 [INFO] (app.py:app:106): Web service listening on port 9004
2022-10-11,22:02:38 [INFO] (web.py:web:2271): 200 GET / (::1) 89.54ms
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Checking if need to handle authentication
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Handling authentication
2022-10-11,22:02:38 [INFO] (base_handler.py:base_handler:115): Authentication is not a required feature for API "v1"
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Checking request headers for authentication data
2022-10-11,22:02:38 [INFO] (base_handler.py:base_handler:115): Authorization header not found
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): authentication not required, username and password are none
2022-10-11,22:02:38 [DEBUG] (state.py:state:511): Collecting Access-Control-Allow-Origin from state file ...
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Origin' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Headers' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Methods' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Description' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Creation Time' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value '0'
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Name' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value 'TabPy Server'
2022-10-11,22:02:38 [INFO] (web.py:web:2271): 200 GET /info (::1) 7.57ms
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Checking if need to handle authentication
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Handling authentication
2022-10-11,22:02:38 [INFO] (base_handler.py:base_handler:115): Authentication is not a required feature for API "v1"
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): Checking request headers for authentication data
2022-10-11,22:02:38 [INFO] (base_handler.py:base_handler:115): Authorization header not found
2022-10-11,22:02:38 [DEBUG] (base_handler.py:base_handler:115): authentication not required, username and password are none
2022-10-11,22:02:38 [DEBUG] (state.py:state:511): Collecting Access-Control-Allow-Origin from state file ...
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Origin' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Headers' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'Access-Control-Allow-Methods' from section [Service Info]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value ''
2022-10-11,22:02:38 [DEBUG] (state.py:state:613): Loading option 'None' from section [Query Objects Service Versions]...
2022-10-11,22:02:38 [DEBUG] (state.py:state:635): Returning value '[]'
2022-10-11,22:02:38 [DEBUG] (state.py:state:148): Collected endpoints: {}
2022-10-11,22:02:38 [INFO] (web.py:web:2271): 200 GET /endpoints (::1) 7.84ms
2022-10-11,22:02:38 [INFO] (web.py:web:2271): 200 GET /tableau.png (::1) 2.55ms
2022-10-11,22:02:38 [WARNING] (web.py:web:2271): 404 GET /favicon.ico (::1) 0.00ms
